packages/client/src/links/HTTPBatchLinkOptions.ts

```diff
@@ -13,6 +14,12 @@ export interface HTTPBatchLinkOptions extends HTTPLinkBaseOptions {
     | ((opts: {
         opList: NonEmptyArray<Operation>;
       }) => HTTPHeaders | Promise<HTTPHeaders>);
+
+  /**
+   * Used for `httpBatchStreamLink`. Will default to the webAPI `TextDecoder`,
+   * but you can use this option if your client runtime doesn't provide it.
+   */
+  textDecoder?: TextDecoderEsque;
```

Comment:
I would much prefer if this was part of **only** the stream options, so we'd have a `HTTPBatchStreamLinkOptions`